src/ring_theory/power_series/well_known.lean

```diff
@@ -73,6 +73,8 @@ variables {A A'} (n : ℕ) (f : A →+* A')
 
 @[simp] lemma coeff_exp : coeff A n (exp A) = algebra_map ℚ A (1 / n!) := coeff_mk _ _
 
+@[simp] lemma constant_coeff_exp : constant_coeff ℚ (exp ℚ) = 1 := rfl
```

Comment:
This should maybe be the more general statement
```
@[simp] lemma constant_coeff_exp : constant_coeff A (exp A) = 1 :=
begin
  rw [constant_coeff, mv_power_series.constant_coeff, ring_hom.coe_mk],
  convert coeff_exp 0,
  { ext, simp, },
  simp,
end
```
but I suspect someone can find a nicer proof than that.

Reply:
Ah, here's the easy proof:
```suggestion
@[simp] lemma constant_coeff_exp : constant_coeff ℚ (exp ℚ) = 1 :=
ring_hom.map_one _
```

Reply:
Thank you!